horovod/tensorflow/mpi_ops.py

```diff
@@ -322,9 +327,12 @@ def _alltoall_grad(op, grad):
     splits = tf.cond(tf.equal(tf.size(splits), 0),
                      lambda : tf.ones([size()], dtype=tf.int32) * (tf.shape(tensor)[0] // size()),
                      lambda : splits)
-    recvsplits = alltoall(splits, splits=[1 for _ in range(size())],
-                          ignore_name_scope=ignore_name_scope)
-    return [alltoall(grad, splits=recvsplits, ignore_name_scope=ignore_name_scope), None]
+    recvsplits = op.outputs[1]
```

Comment:
Great! Reusing the `recvsplits` should make the alltoall gradient computation more efficient.

Reply:
Thanks for the review @romerojosh!